Frameworks/Foundation/NSKVOSwizzling.mm

```diff
@@ -128,12 +128,12 @@ static void notifyingVariadicSetImpl(id self, SEL _cmd, ...) {
     // in the same way: on the stack. For our two supported platforms, this seems to hold true.
     NSMethodSignature* sig = [self methodSignatureForSelector:_cmd];
     auto argSz = [sig getArgumentSizeAtIndex:2];
-    auto nStackArgs = argSz / sizeof(unsigned);
-    unsigned* raw = static_cast<unsigned*>(calloc(sizeof(unsigned), nStackArgs));
+    auto nStackArgs = argSz / sizeof(unsigned int);
+    unsigned int* raw = static_cast<unsigned int*>(calloc(sizeof(unsigned int), nStackArgs));
     va_list ap;
     va_start(ap, _cmd);
     for (unsigned int i = 0; i < nStackArgs; ++i) {
-        raw[i] = va_arg(ap, unsigned);
+        raw[i] = va_arg(ap, unsigned int);
```

Comment:
I thought the NSInvocation wasn't able to support this yet?  If it can I can just swap it out, but fixing NSInvocation is well out of the scope of this PR.